editoast/justfile

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env just --justfile
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Run the application; does not start other services
+run:
+    cargo run
+
+# Install required dependencies
+install:
+    cargo build
+
+# Run the tests
+test:
+    cargo test -- --test-threads=4
+
+# Format the source code
+format:
+    cargo fmt
+
+# Verify code lints without applying any fix
+lint:
+    cargo clippy
```

Comment:
```suggestion
    cargo clippy --workspace --all-targets
```

Reply:
I also added --all-features, as the CI runs that, I hope it’s ok